doc/classes/TreeItem.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TreeItem" inherits="Object" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Control for a single item inside a [Tree].
+		An internal control for a single item inside [Tree].
 	</brief_description>
 	<description>
-		Control for a single item inside a [Tree]. May have child [TreeItem]s and be styled as well as contain buttons.
-		You can remove a [TreeItem] by using [method Object.free].
+		An internal control for a single item inside [Tree]. It can contain other [TreeItem]s as children, be styled, as well as contain buttons.
+		To remove a [TreeItem], you can use use [method Object.free].
```

Comment:
The description gives an impression that it's a Control node, while it isn't. Maybe you could add a mention how to create items?

Reply:
It's a mock-up control... I didn't really know how to describe this intuitively so I didn't change it a ton. Do you have any ideas?

Reply:
```
A single item of a [Tree] control. It can contain other [TreeItem]s as children, be styled, as well as contain buttons. [TreeItem] is not a [Node]; it's created using [method Tree.create_item] or [method TreeItem.create_child].
```
btw not sure what "be styled" part refers to. TreeItem styles are part of Tree. Maybe this part should be removed?